src/components/Container.js

```diff
@@ -67,8 +67,14 @@ const Container = createReactClass({
       this.props.willCreateEditor();
     }
 
-    const mobiledoc = this.props.mobiledoc || EMPTY_MOBILEDOC;
+
+    let mobiledoc = this.props.mobiledoc;
     const { atoms, autofocus, cardProps, cards, html, placeholder, serializeVersion, spellcheck } = this.props;
+
+    if (! mobiledoc && ! html) {
```

Comment:
Could simplify this to a 1 liner: `const mobiledoc = this.props.mobiledoc || (!html && EMPTY_MOBILEDOC)`